core/src/modules/configLoaders/group.ts

```diff
@@ -11,14 +11,15 @@ import { Group } from "../../models/Group";
 import { TopLevelGroupRules } from "../../modules/topLevelGroupRules";
 import { Property } from "../../models/Property";
 import { ConfigWriter } from "../configWriter";
+import { Deprecation } from "../deprecation";
 
 export async function loadGroup(
   configObject: GroupConfigurationObject,
   externallyValidate: boolean,
   validate = false
 ): Promise<IdsByClass> {
   let isNew = false;
-  validateConfigObjectKeys(Group, configObject, ["rules"]);
+  validateConfigObjectKeys(Group, configObject, ["rules", "type"]);
```

Comment:
For now, we won't crash if `group.type` is provided on a config object, but we check it below